ecosystem/sep-0024.md

```diff
@@ -1077,7 +1077,7 @@ Each object in the `transactions` array should have the following fields:
 | `amount_in`               | string              | Amount received by anchor at start of transaction as a string with up to 7 decimals. Excludes any fees charged before the anchor received the funds.                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
 | `amount_in_asset`         | string              | (optional) The asset received or to be received by the Anchor. Must be present if the deposit/withdraw was made using non-equivalent assets. The value must be in [SEP-38 Asset Identification Format](sep-0038.md#asset-identification-format). See the [Asset Exchanges](#asset-exchanges) section for more information.                                                                                                                                                                                                                                                                                             |
-| `amount_out`              | string              | Amount sent by anchor to user at end of transaction as a string with up to 7 decimals. Excludes amount converted to XLM to fund account and any external fees.                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+| `amount_out`              | string              | (optional) Amount sent by anchor to user at end of transaction as a string with up to 7 decimals. Excludes amount converted to XLM to fund account and any external fees. This field is not set only when there is an indicative quote associated with this transaction.                                                                                                                                                                                                                                                                                                                                               |
```

Comment:
The current wording implies the field shouldn't ever be set for transactions using indicative quotes. My suggestion is a bit wordy but I think its important to highlight that the field should always be set eventually.

```suggestion
| `amount_out`              | string              | (optional) Amount sent by anchor to user at end of transaction as a string with up to 7 decimals. Excludes amount converted to XLM to fund account and any external fees. This field should be set as soon as the anchor can calculate it using `amount_in`, the exchange rate between `amount_in_asset` and `amount_out_asset`, and `fee.total`.                                                                                                                                                                                                                                                                                                                                               |
```

Reply:
Thanks!